packages/dcos-integration-test/extra/test_adminrouter_open.py

```diff
@@ -66,7 +67,6 @@ def test_accept_gzip(self, dcos_api_session):
             r.raise_for_status()
             log.info('Response headers: %s', repr(r.headers))
             assert r.headers.get('content-encoding') == 'gzip'
-            assert 'max-age=36000' in r.headers['cache-control']
```

Comment:
So you're saying we do not want to test for the presence of the `cache-control` header in this test, because that mixes unrelated concepts

Reply:
Yes, if it is tested in CI, it should be in a separate test. I didn't add that test, as the main beneficiary is the browser. I manually checked that the pages were cached and refreshed as expected.